easybuild/easyconfigs/s/SALMON/SALMON-1.2.1-intel-2018b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'SALMON'
+version = '1.2.1'
+
+homepage = 'https://salmon-tddft.jp'
+description = """SALMON is an open-source computer program for ab-initio quantum-mechanical calculations of
+ electron dynamics at the nanoscale that takes place in various situations of light-matter interactions.
+ It is based on time-dependent density functional theory, solving time-dependent Kohn-Sham equation in 
+real time and real space with norm-conserving pseudopotentials."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = ['https://github.com/SALMON-TDDFT/SALMON/archive/']
+sources = ['v.%(version)s.tar.gz']
+checksums = ['a5045149e49abe9dd9edefe00cd1508a1323081bc3d034632176b728effdbaeb']
+
+builddependencies = [('CMake', '3.12.1')]
+
+dependencies = [('libxc', '4.2.3')]
+
+separate_build_dir = True
+
+configure_cmd = "python ../SALMON-v.%(version)s/configure.py"
+configopts = "FC='$MPIF90' CC='$MPICC' FFLAGS='$FFLAGS' CFLAGS='$CFLAGS' "
```

Comment:
Wait, this doesn't make sense... If you use single quotes around `$CFLAGS` & co, they won't get expanded by the shell (it'll be a literal `'$CFLAGS'` value).

So, why does this work?! :)

Reply:
you're right... but this are arguments to the `configure.py` script, which then does:

```python
define = ''
for k,v in dict.items():
  define = '{0} -D {1}={2}'.format(define, k, v)

env = ''
for var in args:
  (k, v) = var.split('=', 1)
  env = '{0} {1}="{2}"'.format(env, k, v)

### configuration                                                                                                  
comm = '{2} cmake {0} {1}'.format(define, SOURCE_DIR, env)
print('    $ %s' % comm)
os.system(comm)

```
so in this case I suppose it's actually better that it is not expanded here, only in the `cmake` command executed by the `configure.py` script (?)

Reply:
Ah, that explains it... Thanks for the clarification! (also, ugh...)